cfgov/data_research/blocks.py

```diff
@@ -51,6 +69,31 @@ class ConferenceRegistrationForm(AbstractFormBlock):
         )
     )
 
+    def clean(self, value):
```

Comment:
Unfortunately none of the unit tests cover this new code. I tested this manually and it seems to work but unit tests would be nice for future-proofing.